kmod/src/kernelcompat.h

```diff
@@ -215,4 +215,16 @@ struct kc_shrinker_funcs {
 } while (0)
 #endif
 
+#ifdef KC_USER_MSGHDR
+#define kc_msghdr user_msghdr
+#define kc_kernel_getsockname(a, b, c) kernel_getsockname(a, b)
```

Comment:
Like other wrappers, try to use meaningful argument names here so there's just a wee bit more context for our brains when we're bouncing around the code.  Extra credit for adding nop use of args that are dropped so that they don't trigger unused warnings in older code, but the existing codes pretty spotty with that.. best effort, I guess.